fdbserver/DataDistribution.actor.cpp

```diff
@@ -1321,6 +1324,22 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		addTeam(newTeamServers, isInitialTeam);
 	}
 
+	ACTOR static Future<Void> notifySSTeamChange(Optional<Reference<AsyncVar<struct ServerDBInfo>>> dbInfo)
+	{
+		if (dbInfo.present()) {
+			state Future<ErrorOr<Void>> proxyReply = dbInfo.get()->get().client.commitProxies[0].ssTeamChange.tryGetReply(SSTeamChangeRequest());
+			try {
+				wait(throwErrorOr(proxyReply));
```

Comment:
This probably needs to be in choose when block, with one block to handle case where dbInfo changes. You can find examples by looking for `dbInfo->onChange` inside `when` block.